src/main/java/core/basesyntax/impl/WriteToCsv.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax.impl;
+
+import core.basesyntax.service.WriteService;
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.List;
+
+public class WriteToCsv implements WriteService {
+    @Override
+    public void writeToFile(List<String> preparedData, String path) {
```

Comment:
Single responsibility violation. This class not only write something to file but also handles convetion of list to string wich /n. This method should revieve String report, String path And just write report to file